packages/manager/apps/pci-block-storage/src/api/hooks/useCatalogWithPreselection.tsx

```diff
@@ -35,7 +35,11 @@ export const useCatalogWithPreselection = (
     { data: catalogData, ...restCatalogQuery },
   ] = useQueries({
     queries: [
-      getVolumeQuery(projectId, volumeId),
+      {
+        ...getVolumeQuery(projectId, volumeId),
+        refetchOnWindowFocus: true,
+        refetchOnMount: true,
+      },
```

Comment:
no need for `refetchOnReconnect` here?

Reply:
Ok i got confused, we actually don't need to refresh here